packages/the-org/src/communityManager/index.ts

```diff
@@ -403,6 +405,133 @@ const config: OnboardingConfig = {
   },
 };
 
+
+/**
+ * Test suite for the community Manager agent
+ * Tests onboarding and cross-platform functionality
+ */
+
+export class CommunityManagerTestSuite implements TestSuite {
```

Comment:
can we move this to another test file and not have the test logic in the communityManager index.ts